src/form/Input.js

```diff
@@ -45,7 +45,7 @@
                 break;
             case "checkbox":
                 this.checkNodeName("INPUT", element);
-                this._inputElement.property("checked", this.value());
+                this._inputElement.property("checked", this.value() === "true");
```

Comment:
@GordonSmith The **value** for a checkbox needs to be evaluated as a string because src/api/IInput.js stores the **.value()** as a "string" publish parameter.